### PR TITLE
Backport neutron mlnx agent

### PR DESCRIPTION
**Pull request by brtkwr**
```
git review -X I5a5bd255b937fe5d4c56cc21ef898b796c5bda41
```

### CODE REVIEW

File: ansible/kolla-ansible.yml [hunk 1]
```diff
@@ -163,151 +319,12 @@
...
-        # While kayobe has its own support for installing an NTP daemon, the
-        # kolla-ansible baremetal role does a one-time sync which is useful.
-        kolla_enable_host_ntp: "{{ ntp_service_enabled }}"
-        docker_daemon_mtu: "{{ public_net_name | net_mtu | default }}"
+        kolla_ansible_pass_through_host_vars: "{{ kolla_overcloud_inventory_pass_through_host_vars }}"
+        kolla_ansible_pass_through_host_vars_map: "{{ kolla_overcloud_inventory_pass_through_host_vars_map }}"
+        kolla_ansible_inventory_path: "{{ kolla_config_path }}/inventory/overcloud"
+        # Kolla ansible expects these variables to be comma-separated lists.
+        kolla_neutron_bridge_names: "{{ kolla_neutron_bridge_names | join(',') }}"
+        kolla_neutron_external_interfaces: "{{ kolla_neutron_external_interfaces | join(',') }}"
```
Comment (markgoddard) [inline ansible/kolla-ansible.yml]: What this part a clean merge?

Comment (brtkwr) [reply ansible/kolla-ansible.yml]: Yes, these came from upstream and not my cherry-picks

**Review by jovial — APPROVED** [review]
Looks reasonable to me - Assuming the merge was clean, so only verified the cherry-pick.

**Comment by markgoddard** [comment]
Github says there are merge conflicts

**Comment by brtkwr** [comment]
> Github says there are merge conflicts

Do you might resolving the here: https://github.com/stackhpc/kayobe/pull/203/conflicts? I am not 100% sure whats needed.

**Comment by brtkwr** [comment]
My gut feeling is to keep everything.

**Comment by brtkwr** [comment]
i propose merging the single cherry-pick for now

**Comment by markgoddard** [comment]
> i propose merging the single cherry-pick for now

I generally prefer to split pure upstream merges from backports. Backports should get extra attention.

**Review by markgoddard — APPROVED** [review]

**Comment by markgoddard** [comment]
Please also backport to Ussuri.

**Comment by brtkwr** [comment]
I forked stable/ussuri and cherry-picked the change before pushing the branch: https://github.com/stackhpc/kayobe/tree/stackhpc/ussuri

**Pull request merged**